TypeCobol/AntlrGrammar/CobolCodeElements.g4

```diff
@@ -4433,6 +4434,21 @@ exitMethodStatement:
 exitParagraphStatement:
 	EXIT ({ string.Equals(CurrentToken.Text, "PARAGRAPH", System.StringComparison.OrdinalIgnoreCase) }? KeywordPARAGRAPH=UserDefinedWord);
 
+// EXIT PERFORM statement
+// The EXIT PERFORM statement controls the exit from an inline PERFORM without using a GO TO statement
+// or a PERFORM ... THROUGH statement.
+// If you specify an EXIT PERFORM statement outside of an inline PERFORM statement, the EXIT PERFORM
+// is ignored.
+// When an EXIT PERFORM statement without the CYCLE phrase is executed, control is passed to an implicit
+// CONTINUE statement. This implicit CONTINUE statement immediately follows the END-PERFORM phrase
+// that matches the most closely preceding and unterminated inline PERFORM statement.
+// When an EXIT PERFORM statement with the CYCLE phrase is executed, control is passed to an implicit
+// CONTINUE statement. This implicit CONTINUE statement immediately precedes the END-PERFORM
+// phrase that matches the most closely preceding and unterminated inline PERFORM statement.
+
+exitPerformStatement:
+	EXIT PERFORM cycle?;
```

Comment:
Ok the `CYCLE` contetxual keyword is already defined for SQL, could you move the definition here ?. I think it is cleaner.

Reply:
Done